test-requirements.txt

```diff
@@ -0,0 +1 @@
+freezegun==0.3.11
```

Comment:
Do we really need to pin a specific version? If a minimal version is required, the dependency must be delcared as 'freezegun>=0.3.11`  otherwise we should remove the version part of the dependency declaration `freezegun`

Reply:
More recent `freezegun` fails

Reply:
in such a case you can specify the dependency as follow `freezegun<=0.3.11` and add a comment to explain that tests are failing with freezegun> 3.0.11.
```
# Tests fails with freezegun > 3.0.11
freezegun <= 3.0.11
```
@simahawk Are you aware of some problems with the latest version of `freezegun`.

Reply:
```
"/home/travis/build/brainbeanapps/social/setup/mail_activity_reminder/odoo/addons/mail_activity_reminder/tests/test_mail_activity_reminder.py", line 6, in <module>
    from freezegun import freeze_time
  File "/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/freezegun/__init__.py", line 9, in <module>
    from .api import freeze_time
  File "/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/freezegun/api.py", line 26, in <module>
    _EPOCHTZ = datetime.datetime(1970, 1, 1, tzinfo=dateutil.tz.UTC)
AttributeError: module 'dateutil.tz' has no attribute 'UTC'
```

Reply:
nope AFAIK